plugins/GSdx/GSSettingsDlg.cpp

```diff
@@ -320,6 +322,13 @@ void GSSettingsDlg::UpdateRenderers()
 	{
 		GSRendererType ini_renderer = GSRendererType(theApp.GetConfigI("Renderer"));
 		renderer_setting = (ini_renderer == GSRendererType::Undefined) ? GSUtil::GetBestRenderer() : ini_renderer;
+		
+		if (ini_renderer == GSRendererType::Undefined) {
+			if (renderer_setting == GSRendererType::OGL_HW)
+				ComboBoxInit(IDC_CRC_LEVEL, theApp.m_gs_crc_level, 2);
```

Comment:
I had used SendMessge in an early draft of the commit, but it didn't work correctly. It must've been some mistake I made, because it seems to works perfectly now. :+1: